### ISSUE TITLE
Update Bootstrap

### ISSUE THREAD

**Issue by kgokdemir** [xerocross/vue-project-template#1]
Thanks for sharing. By the way since you're template has Bootstrap version 3.7.1. 
You should update it to 4.3.1 version for newer styles and more modern appearance.

**Comment by xerocross** [comment]
Yeah.  Fair point.  I have moved most of my individual projects over to Bootstrap 4.  I don't think there is anything in this template that actually relies on Bootstrap either way.  It's easy enough to take it out or update to the latest one.
